### DISCUSSION TITLE
Documentation Apple Beacons

### DISCUSSION THREAD

**Discussion by SamB-GB** [ESPresense/ESPresense#797] [Support]
As it is right now I need to add the IRK to the known devices list on each ESPresence.

However in the documentation it states:

Tip: use MQTT explorer to view the RAW messages. Note: With the automatic procedure you DON’T need to add the 32-bit key to the Known BLE identity resolving keys. It’s automatically done via mqtt!

What is the automatic procedure exactly?

**Comment by pyrodex** [comment]
I didn't have to add the IRK to each ESPresense node. When I enrolled one of my devices it wrote to the MQTT server and it seems all of them consume that as a common source for reference. That has been my experience since deploying from the beginning on 3.1.1.

**Comment by Terastar-Paperclip** [comment]
This is listed as an active docs gap on our enrollment reference (#2324) — "Apple Beacons documentation." Linking so it stays visible.
